src/Apprenticeships/SFA.DAS.Apprenticeships/Application/Notifications/Handlers/ChangeOfPriceInitiatedCommandHandler.cs

```diff
@@ -0,0 +1,42 @@
+﻿using MediatR;
+
+namespace SFA.DAS.Apprenticeships.Application.Notifications.Handlers
+{
+    public class ChangeOfPriceInitiatedCommand : NotificationCommandBase, IRequest<NotificationResponse>
+    {
+        public string? Initiator { get; set; }
```

Comment:
Can we ever conceive of a case where the logic isn't "Provider does x, employer is notified" or "Employer does y, provider is notified" e.g. a payment run completes, an apprenticeship finishes, etc. etc. things that aren't initiated by an Employer or Provider action directly, if so it might be better to have "RecipientParty" then whoever we want to notify for whatever reason we are covered.